src/common/utils.js

```diff
@@ -278,6 +278,13 @@ export const formatSpeed = (bytes, milliseconds) => humanFormat(
   { scale: 'binary', unit: 'B/s' }
 )
 
+export const formatDuration = nbSecondes => _('formatDuration', {
```

Comment:
The name is incorrect, this function does not format a duration, it parse a duration in multiple units.